pkg/apis/kops/componentconfig.go

```diff
@@ -318,6 +318,10 @@ type KubeAPIServerConfig struct {
 	// OIDCClientID is the client ID for the OpenID Connect client, must be set
 	// if oidc-issuer-url is set.
 	OIDCClientID *string `json:"oidcClientID,omitempty" flag:"oidc-client-id"`
+	// A key=value pair that describes a required claim in the ID Token.
+	// If set, the claim is verified to be present in the ID Token with a matching value.
+	// Repeat this flag to specify multiple claims.
+	OIDCRequiredClaim map[string]string `json:"oidcRequiredClaim,omitempty" flag:"oidc-required-claim"`
```

Comment:
Has this been tested with multiple claims?

The documentation 
```shell
--oidc-required-claim mapStringString
  | A key=value pair that describes a required claim in the ID Token. If set, the claim is verified to be present in the ID Token with a matching value. Repeat this flag to specify multiple claims.
```
i.e. the `Repeat this flag to specify multiple claims` ... but the default of the [flagbuilder reflection](https://github.com/kubernetes/kops/blob/master/pkg/flagbuilder/build_flags.go#L96-L115) is to append `map[string]string like --node-labels=k1=v1,k2=v2 etc`

If it's not supported perhaps use a [slice](https://github.com/kubernetes/kops/blob/master/pkg/flagbuilder/build_flags.go#L119-L138) and add the `repeat` tag to the annotation

Reply:
Yes, the flag doesn't support multiple claims. Thanks for the suggestion, that makes sense. Will try 👍